### PR TITLE
[master] Fixed: Configuration value '__utils__' needs to be set when creating …

### PR DESCRIPTION
**Pull request by KarlAustin**
…OpenNebula VM

### What does this PR do?
It fixes an issue when creating VMs for OpenNebula Cloud driver.

### What issues does this PR fix or reference?
Fixes: https://github.com/saltstack/salt/issues/64577

### Previous Behavior
VM creation fails with error

### New Behavior
VM creation proceeds

### Merge requirements satisfied?
- [n/a] Docs
- [x] Changelog - https://docs.saltproject.io/en/master/topics/development/changelog.html
- [see below] Tests written/updated

Currently there is no test existing for create() for the OpenNebula driver.  This situation has not changed.

### Commits signed with GPG?
No

### CODE REVIEW

**Review by twangboy — DISMISSED** [review]

**Review by dmurphy18 — DISMISSED** [review]

**Comment by twangboy** [comment]
@KarlAustin Would you mind fixing the pre-commit failures?

**Review by garethgreenaway — DISMISSED** [review]

**Comment by KarlAustin** [comment]
> @KarlAustin Would you mind fixing the pre-commit failures?

Assuming I'm reading it right, these all look to be related to string formatting in code I've not touched? 

More than willing to bring them up to date in a separate PR though, which would seem to make sense as they aren't really related to this issue.

**Comment by nicholasmhughes** [comment]
@KarlAustin the string formatting is due to recent changes in the pyupgrade pre-commit check. If you merge the latest master branch and `pre-commit run --files [files you've changed]`, the pre-commit checks will take care of the changes for you. Then, just push the changed files up and everything will be happy.

**Review by twangboy — APPROVED** [review]

File: salt/cloud/clouds/opennebula.py [hunk 1]
```diff
@@ -1077,7 +1071,7 @@ def __query_node_data(vm_name):
             return node_data
 
     try:
-        data = __utils__["cloud.wait_for_ip"](
+        data = salt.utils.cloud.wait_for_ip(
```
Comment (Ch3LL) [inline salt/cloud/clouds/opennebula.py:1074]: I'm not seeing `salt.utils.cloud` imported into this module

**Review by Ch3LL — CHANGES_REQUESTED** [review]

**Comment by Ch3LL** [comment]
@KarlAustin did you see my review comment?

**Comment by dwoz** [comment]
Closing this due to inactivity. Anyone should feel free to re-open it if they want to see it through to the end in one release cycle.

@KarlAustin If you'd like to get this change into a release please re-open it after re-basing.

**Pull request closed without merging**